.github/linters/.hadolint.yaml

```diff
@@ -2,3 +2,4 @@ ignored:
   # it's good to be in sync with the latest versions of the tools and only pin versions in specific cases
   - DL3008 # warning: Pin versions in apt get install.
   - DL3013 # warning: Pin versions in pip.
+  - DL3059 # info: Multiple consecutive `RUN` instructions.
```

Comment:
Isn't this specific for Dockerfiles?

Reply:
Yep. The GH Actions updates I brought from https://github.com/orise-robotics/ros_ws/pull/59